tools/bin/load_test/source_spec.json

```
@@ -0,0 +1,17 @@
+{
+  "name": "End-to-End Testing (Mock API)",
+  "sourceDefinitionId": "50bd8338-7c4e-46f1-8c7f-3ef95de19fdd",
```

if these are arbitrary I like to give clues to that in the example itself for debugging
```suggestion
  "sourceDefinitionId": "abc12345-test-code-test-SourceDefId1",
```

---

ah that was supposed to be a variable and be replaced by script (after fetching the sourceDefinitionId from API put the result here). Latest commit have had this corrected!